packages/localize-tools/src/tests/transform.unit.test.ts

```diff
@@ -353,6 +353,28 @@ test('msg(string(<b>msg(string)</b>)) translated', () => {
   );
 });
 
+test('html(msg(string)) with msg as attr value', () => {
```

Comment:
Could you add one with a `.bar` property attribute, too? Same result, just for completeness.

Reply:
Done, and also added additional tests for having multiple attributes in a single template expression so as to test the logic for looking at both the head and the middle.

Reply:
Nice!